tests/uncoupled_two_phase_tian_parameterization_kinematic_slab.prm

```diff
@@ -0,0 +1,55 @@
+# This test is for the fully coupled Simplified Subduction Model with Parameterized
+# Solid-Fluid Reactions cookbook. It also doubles as a test for the min/max
+# compaction viscosity.
```

Comment:
This does not have melt transport, so it does not have compaction viscosities, right?